### PR TITLE
enhance: clean compile warnings

### PR DESCRIPTION
**Pull request by RCmerci**

### CODE REVIEW

File: lib/export/conf.ml [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+  | _ -> Error "invalid format"
+
 type t = {
     (* html: bool; *)
     (* hiccup: bool; *)
     toc: bool;
     heading_number: bool;
     keep_line_break: bool;
-    format: string;
+    format: format;
```
Comment (tiensonqin) [inline lib/export/conf.ml:19]: It's much better now.

File: lib/parsers.ml [hunk 2]
```diff
@@ -10,7 +10,7 @@ let is_tab = function
   | _ -> false
 
 let is_tab_or_space = function
-  | '\t' | ' ' | '\026' | '\012' | '\009'   -> true
+  | '\t' | ' ' | '\026' | '\012' -> true
```
Comment (tiensonqin) [inline lib/parsers.ml:13]: What's the reason to remove `\009`? It's the horizontal tab, right?

Comment (RCmerci) [reply lib/parsers.ml:13]: \009 = '\t' 
image

**Comment by RCmerci** [comment]
@tiensonqin Any more problems with this PR?

File: lib/parsers.ml (hunk 2 above)
Comment (tiensonqin) [reply lib/parsers.ml:13]: ah, got it

**Comment by tiensonqin** [comment]
Nope!

**Pull request merged**